test/nxos/mocked_data/test_get_lldp_neighbors_detail/normal/expected_result.json

```diff
@@ -1,37 +1,55 @@
 {
-    "Eth1/1": [
+    "Ethernet1/1": [
         {
             "remote_port_description": "Ethernet1/1", 
             "remote_port": "Ethernet1/1", 
             "remote_system_description": "Cisco Nexus Operating System (NX-OS) Software 7.0(3)I4(1)", 
-            "remote_chassis_id": "50:87:89:A1:D8:D6", 
-            "remote_system_enable_capab": "B, R", 
+            "remote_chassis_id": "5087.89a1.d8d6",
+            "remote_system_enable_capab": [
+                "bridge",
+                "router"
+            ],
             "parent_interface": "", 
-            "remote_system_capab": "B, R", 
+            "remote_system_capab": [
+                "bridge",
+                "router"
+            ],
             "remote_system_name": "n9k2.ntc.com"
         }
     ], 
     "mgmt0": [
         {
             "remote_port_description": "FastEthernet1/0/9", 
-            "remote_port": "FastEthernet1/0/9", 
+            "remote_port": "Fa1/0/9",
```

Comment:
This seems a bit strange? Why did this go to an abbreviated name?

Reply:
That’s how it’s in the mock data: 
https://github.com/napalm-automation/napalm/blob/develop/test/nxos/mocked_data/test_get_lldp_neighbors_detail/normal/show_lldp_neighbors_detail.json#L8

This is a fix for #771